docs/learn-governance.md

```diff
@@ -281,30 +286,40 @@ support - will move to a public referendum under a neutral, majority-carries vot
 case that all members of the council vote in favor of a motion, the vote is considered unanimous and
 becomes a referundum with negative adaptive quorum biasing.
 
-A two-thirds majority of the council can cancel a referendum. This may function as a last-resort if
-there is an issue found late in a referendum's proposal such as a bug in the code of the runtime
-that the proposal would institute.
+### Canceling
+
+A proposal can be canceled if the [technical committee](#technical-committee) unanimously agrees to
+do so, or if Root origin (sudo or Council) triggers this functionality. A canceled proposal's
+deposit is burned.
+
+Additionally, a two-thirds majority of the council can cancel a referendum. This may function as a
+last-resort if there is an issue found late in a referendum's proposal such as a bug in the code of
+the runtime that the proposal would institute.
 
 If the cancellation is controversial enough that the council cannot get a two-thirds majority, then
 it will be left to the stakeholders _en masse_ to determine the fate of the proposal.
 
-> For information on council, please head over to our
-> [technical explainer video](https://www.youtube.com/watch?v=837Vv3gdRzI&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=32&ab_channel=Polkadot).
+### Blacklisting
+
+A proposal can be blacklisted by Root origin (sudo or Council). A blacklisted proposal and its
+related referendum (if any) is immediately [canceled](#canceling). Additionally, a blacklisted
+proposal's hash can never re-appear in the proposal queue.
 
 ### How to be a council member?
 
 ![](assets/governance/approval-vote.png)
 
-Currently there are 13 seats on the council. All stakeholders are free to signal their approval of
-any of the registered candidates.
+All stakeholders are free to signal their approval of any of the registered candidates.
 
 Council elections are handled by the same [Phragmén election](learn-phragmen) process that selects
 validators from the available pool based on nominations. However, token holders' votes for
 councillors are isolated from any of the nominations they may have on validators. Council terms last
-for one day. At the end of each term, Phragmén election algorithm runs and the result will choose
-the new councillors based on the vote configurations of all voters. The election also chooses a set
-number of runners up (currently seven on Kusama and fifteen on Polkadot) that will remain in the
-queue with their votes intact.
+for one day on Kusama and one week on Polkadot.
+
+At the end of each term, Phragmén election algorithm runs and the result will choose the new
+councillors based on the vote configurations of all voters. The election also chooses a set number
+of runners up (currently 19 on Kusama and 20 on Polkadot) that will remain in the queue with their
```

Comment:
If these values are changing often too, would the injected script be a better fit here?

Reply:
They will not change often, but yes this is ripe for injection later on.